src/app/models/hand.ts

```diff
@@ -8,27 +8,30 @@ export class Hand {
         this.color = color;
     }
 
-    async load(): Promise<THREE.Object3D> {
-        const mtlLoader = new MTLLoader();
-        const materials = await new Promise(resolve => {
-            mtlLoader.load('assets/models/hand.mtl', mat => {
-                resolve(mat);
-            });
-        });
-
-        return new Promise(res => {
-            const objLoader = new OBJLoader();
-            objLoader.setMaterials(materials);
-            objLoader.load(
-                './assets/models/hand.obj',
-                (obj) => {
-                    const hand = obj.getObjectByName('hand');
-                    hand.scale.set(3, 3, 3);
-                    hand.material.color.setHex(this.color);
+    get obj(): Promise<THREE.Object3D> {
```

Comment:
This method is not used, remove. Let us use `load` instead of wrapping in with `get`. This was a bad idea on my part.

Reply:
See a596abe for the changes.